data/intro missions.txt

```diff
@@ -449,5 +449,42 @@ conversation "end of intro missions"
 		goto next
 	
 	label next
-	`	There's an awkward pause. You're unsure of how to respond. Then he says, "Thing is, this here planet is one of the most exclusive retirement communities in the galaxy. Plenty of the locals are probably worth more than a billion credits. But they're mostly financial or business types, or old money. I'm rich enough to be here, but I just hope they're willing to socialize with someone who made his fortune flying a ship."`
-	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area."`
+	`	There's an awkward pause. You're unsure of how to respond. Then he says, "Thing is, this here planet is one of the most exclusive retirement communities in the galaxy. Plenty of the locals are probably worth more than a billion credits. But they're mostly financial or business types, or old money. I'm rich enough to be here, but I just hope they're willing to socialize with someone who made his fortune flying a ship.`
+	`	"Before you go, is there anything else you'd like to ask me about being a new pilot?"`
+	label choices
+	choice
+		`	"Where in space should I go if I want to avoid pirates?"`
```

Comment:
`Where in space should I go` -> `Where should I go` or `Which systems should I keep to` or something like that